distelli-manifest.yml

```diff
@@ -0,0 +1,25 @@
+team-modules/puppetlabs-vsphere:
```

Comment:
We're doing both pipelines and GHA for the time being, I take it?

Reply:
It should be removed.